x/attribute/keeper/migrations.go

```diff
@@ -17,6 +17,8 @@ func NewMigrator(keeper Keeper) Migrator {
 
 // Migrate1to2 migrates from version 1 to 2 to convert attribute keys from 20 to 32 length
 func (m *Migrator) Migrate1to2(ctx sdk.Context) error {
-	ctx.Logger().Info("Migrating Attribute Module from Version 1 to 2")
-	return v042.MigrateAddressLength(ctx, m.keeper.storeKey)
+	ctx.Logger().Info("Migrating Attribute Module from Version 1 to 2 (1/1)")
+	err := v042.MigrateAddressLength(ctx, m.keeper.storeKey)
+	ctx.Logger().Info("Finished Migrating Attribute Module from Version 1 to 2")
```

Comment:
#nit Not important but instead of creating a new variable and then returning it after the logging statement, would it be a little clearer to put the logging in a defer statement?
```golang
ctx.Logger().Info("Migrating Attribute Module from Version 1 to 2 (1/1)")
defer ctx.Logger().Info("Finished Migrating Attribute Module from Version 1 to 2")
return v042.MigrateAddressLength(ctx, m.keeper.storeKey)
```

Reply:
Good point, I didn't think of that.  I will use something like this next time.  Since, if I were to change it now.  I would feel like I need to run the tests again which takes hours.